device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/port_config.ini

```diff
@@ -0,0 +1,21 @@
+# name         lanes             alias        index  role       speed       asic_port_name     coreId      corePortId      numVoq
+Ethernet144    72,73,74,75       Ethernet19/1 19     Ext        100000      Eth144-ASIC1       1           1               8
```

Comment:
The 'asic_port_name' should start from Eth0 again.

Reply:
Fixed